prometheus/config/args/install.sls

```diff
@@ -108,8 +108,13 @@ prometheus-config-args-{{ name }}-all:
 prometheus-config-args-{{ name }}-file-managed:
   file.managed:
     - name: {{ args_file }}
+    {% if grains['os_family'] not in ['Gentoo']  %}
     - contents: |
         ARGS="{{ concat_args(args) }}"
+    {% else %}
+    - contents: |
+        command_args='{{ concat_args(args) }}'
+    {% endif %}
```

Comment:
We could lose this `if` block by providing `ARGS` / `command_args` via. the map.  So the former in `defaults.yml` and the latter in `osfamilymap.yaml`, with the final solution looking something like:

```sls
    - contents: |
        {{ prometheus.config.args_setting }}="{{ concat_args(args) }}"
```